contribute-code.md

```diff
@@ -0,0 +1,296 @@
+
+To make the changes easily mergeable, please rebase them to the latest version of the main repositories master branch. Please do also respect the [commit message guidelines]( {{ site.base }}/contribute-code.html#coding-guidelines ), clean up your commit history, and squash your commits to an appropriate set. Please verify your contribution one more time after rebasing and commit squashing as described above.
+
+The Flink project accepts code contributions through the [GitHub Mirror](https://github.com/apache/flink), in the form of [Pull Requests](https://help.github.com/articles/using-pull-requests). Pull requests are a simple way to offer a patch, by providing a pointer to a code branch that contains the change.
+
+To open a pull request, push our contribution back into your fork of the Flink repository.
+
+```
+git push origin myBranch
+```
+
+Go the website of your repository fork (`https://github.com/<your-user-name>/flink`) and use the *"Create Pull Request"* button to start creating a pull request. Make sure that the base fork is `apache/flink master` and the head fork selects the branch with your changes. Give the pull request a meaningful description and send it.
+
+It is also possible to attach a patch to a [JIRA]({{site.FLINK_ISSUES_URL}}) issue.
+
+-----
+
+## Coding guidelines
+
+### Pull requests and commit message
+{:.no_toc} 
+
+- **Single change per PR**. Please do not combine various unrelated changes in a single pull request. Rather, open multiple individual pull requests where each PR refers to a JIRA issue. This ensures that pull requests are *topic related*, can be merged more easily, and typically result in topic-specific merge conflicts only.
+
+- **No WIP pull requests**. We consider pull requests as requests to merge the referenced code *as is* into the current *stable* master branch. Therefore, a pull request should not be "work in progress". Open a pull request if you are confident that it can be merged into the current master branch without problems. If you rather want comments on your code, post a link to your working branch.
+
+- **Commit message**. A pull request must relate to a JIRA issue; create an issue if none exists for the change you want to make. The latest commit message should reference that issue. An example commit message would be *[FLINK-633] Fix NullPointerException for empty UDF parameters*. That way, the pull request automatically gives a description of what it does, for example what bug does it fix in what way.
+
+- **Append review commits**. When you get comments on the pull request asking for changes, append commits for these changes. *Do not rebase and squash them.* It allows people to review the cleanup work independently. Otherwise reviewers have to go through the entire set of diffs again.
+
+### Exceptions and error messages
+{:.no_toc} 
+
+- **Exception swallowing**. Do not swallow exceptions and print the stacktrace. Instead check how exceptions are handled in similar situations.
```

Comment:
That would depend on the code where the exception should be handled, e.g., a test, the API, runtime, etc.
But I agree, I will try to rephrase the statement.